examples/seismic/viscoacoustic/operators.py

```diff
@@ -6,9 +6,9 @@
 from examples.seismic import PointSource, Receiver
 
 
-def blanch_symes(model, geometry, v, p, **kwargs):
+def sls_1st_stencil(model, geometry, v, p, **kwargs):
```

Comment:
What's the significance behind the `1st` here? Terminology from one of the references?